### PR TITLE
Step2

### PR DESCRIPTION
**Pull request by ArranHL**
### Step in progress

Step 2's objective is to have the VI's:
- Find the RPi with the SSH user
- Log in by Pi by the SSH key
- Add the Key to Github
- Add the Key to treehouses
- Add the key on RPI
- Log in by Root
- Rename Pi by Github Name
- Turn on Tor
- Turn on Tor Notice

I was going to have the VI's Enable SSH on your Raspberry Pi via a different way but thought that perhaps the simplest way may be the best.
There also needs to be a discussion about whether to make SSHing into the Pi more general for the sake of simplicity

### CODE REVIEW

**Comment by dogi** [comment]
@ArranHL ssh is already activated on treehouses image

Step 2's objective is to have the VI's:
- Log in by Pi user with password
- Rename Pi by Github Name
- create sshkeys
- Add the Key to Github
- Add the key on RPI by treehouses command
- Log in by Root
- Turn on Tor
- Turn on Tor Notice

File: pages/vi/firststeps.md [hunk 1]
```diff
@@ -145,26 +145,28 @@ In this step you will:
 - [Add the SSH key to your GitHub account](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account)
 
 
-### Find the RPi with the SSH user
-To connect to Pi via SSH you will need to know your Raspberry Pi's local IP address. If you are running the Pi without a screen, you can find the IP address by opening your terminal and type `nmap -sP 192.168.1.*` to find your raspberri Pi's IP address 
+### Log in with the Pi user
 
+Once you start up your Raspberry Pi and find its local IP address, you can connect to it from your computer. Windows users can use an SSH client like PuTTY.
```
Comment (dogi) [inline pages/vi/firststeps.md]: git bash

File: pages/vi/firststeps.md [hunk 2]
```diff
@@ -145,26 +145,28 @@ In this step you will:
...
 ### Log in by Root
 
+To log into root, you can run `sudo -s` while logged into the Pi user, or ssh directly into it by running `ssh root@[local IP address]` instead of `ssh pi@[local IP address]`.
+
 ### Rename Pi by Github Name
 
+To rename your Raspberry Pi, run `treehouses rename "username"` and replace `username` with your Github username.
+
 ## Tor
 Tor is a computer network run by volunteers worldwide. Each volunteer runs what is called a relay, which is just a computer that runs software allowing users to connect to the Internet via the Tor network.
```
Comment (dogi) [inline pages/vi/firststeps.md]: we do not run a relay ... we are running a hidden service

**Comment by dogi** [comment]
https://raw.githack.com/treehouses/treehouses.github.io/Step2/#!pages/vi/sshpi.md

**Review by dogi — APPROVED** [review]

**Review by Liew211 — APPROVED** [review]

**Review by vers4ce — APPROVED** [review]

**Pull request merged**